sui-execution/latest/sui-adapter/src/execution_engine.rs

```diff
@@ -996,6 +1001,29 @@ mod checked {
         builder
     }
 
+    fn setup_bridge_create(
+        mut builder: ProgrammableTransactionBuilder,
+    ) -> ProgrammableTransactionBuilder {
+        let bridge_uid = builder.programmable_move_call(
+            SUI_FRAMEWORK_ADDRESS.into(),
+            ident_str!("object").to_owned(),
+            ident_str!("bridge").to_owned(),
+            vec![],
+            vec![],
+        );
+        // Set chain id to sui devnet
+        // TODO: set this according to ChainIdentifier
```

Comment:
is this comment true anymore?

Reply:
yes. different chain needs different bridge chain ids

Reply:
ah got it!